orders/views.py

```diff
@@ -1,3 +1,102 @@
-from django.shortcuts import render
+import json
+from enum                 import Enum
 
-# Create your views here.
+from django.views         import View
+from django.http          import JsonResponse
+from django.db            import transaction
+from django.db.models     import Sum, F
+from django.db.utils      import IntegrityError
+
+from orders.models        import Order, OrderItem, OrderStatus, OrderItemStatus, PaymentMethod
+from users.models         import Cart
+from users.utils          import login_decorator
+
+class OrderStatusEnum(Enum):
+    CONFIRMING      = 1
+    PAID            = 2
+    PENDING         = 3
+    DELIVERD        = 4
+    ORDER_CANCELLED = 5
+    REFUNDED        = 6
+
+class OrderItemStatusEnum(Enum):
+    PAID            = 1
+    PREPARING       = 2
+    SHIPPED         = 3
+    DELIVERD        = 4
+    ORDER_CANCELLED = 5
+    REFUNDED        = 6
+    RETURNED        = 7
+
+class OrderView(View):
+    @login_decorator
+    def post(self,request):
+        try:
+            with transaction.atomic():
+                data           = json.loads(request.body)
+                user           = request.user
+                carts          = Cart.objects.filter(user_id = user.id).select_related("user","product")
+                payment_method = data.get('payment')
+                
+                order = Order.objects.create(
+                    user            = user,
+                    status          = OrderStatus.objects.get(id = OrderStatusEnum.CONFIRMING.id),
```

Comment:
Enum을 사용하게되면 정규화된 데이터의 값을 가져오기 위해서 DB를 호출할 필요가 없습니다.
아래와 같이 OrderStatus 객체에서 값을 가져와서 바로 할당하도록 수정해주세요~
```suggestion
                    status = OrderStatusEnum.CONFIRMING.value,
```

Reply:
피드백 반영이 잘 안된 것 같습니다.
`OrderStats.objects.get() `을 사용하지 않고 `OrderStatusEnum.CONFIRMING.value`만 사용하시라는 얘기였습니다!

Reply:
수정 완료했습니다